source/agora/common/crypto/Key.d

```diff
@@ -320,14 +423,19 @@ public struct Seed
         return Base32.encode(bin).assumeUnique;
```

Comment:
This needs fixing

Reply:
Should I remove it, since it's a duplicate of `PrintMode.Clear`?